### PR TITLE
Steering & Technical Committee charter

### PR DESCRIPTION
**Pull request by msfussell**
This is the bootstrap for the Technical and Steering Committee for Dapr

### CODE REVIEW

**Review by artursouza — APPROVED** [review]
It has addressed the comments I made offline. LGTM

File: steering-and-technical-committee-charter.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+
+Public meetings are recorded and the [recordings](XX) and [meeting notes](XXX) are available publicly. If a member misses more than 50% of all meetings held within a six month period then they will be removed from the committee and a special election will be held. 
+
+Questions and proposals for changes to governance are posted as [issues in the community repository](https://github.com/Dapr/community/issues), and the STC invites your feedback there. See [Getting in touch](#getting-in-touch) for other options.
+
+## Committee members
+To bootstrap the STC 5 individuals were identified to provide the initial process that can bootstrap the remainder of the process.The bootstrap committee will be replaced by an elected Dapr Committee. A committee member is given [owner role](https://github.com/orgs/dapr/people?query=role%3Aowner) in the Dapr repository.
```
Comment (yaron2) [inline steering-and-technical-committee-charter.md:70]: Change `A committee member is given [owner role](https://github.com/orgs/dapr/people?query=role%3Aowner) in the Dapr repository` to `A committee member is given [owner role](https://github.com/orgs/dapr/people?query=role%3Aowner) in the Dapr org`.

File: steering-and-technical-committee-charter.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+pass with at least half the members of the committee supporting it.
+
+Quorum is considered reached when at least half of the members are present.
+
+## Composition
+Seats on the steering and technical committee are held by an individual, not by their employer. This means that if an individual changes their employer, they retain their seat until the next election. 
+
+The steering and technical committee has 5 seats initially to bootstrap. These 5 seats are contributing seats, open to any project contributor. This is open to expand upto 11 seats for qualifying contributing members as the project expands. There is no timeframe for this expanded committee. 
+
+A maximum of two seats is allowed from any one company. For companies with two seats these are should be filled by one person focused on the vision, values, and mission of the project and another person who is focused on the technical decisions.
```
Comment (yaron2) [inline steering-and-technical-committee-charter.md:98]: Typo: `For companies with two seats these are should be filled`

**Review by yaron2 — CHANGES_REQUESTED** [review]

File: steering-and-technical-committee-charter.md [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+
+- Making all announcements associated with the election
+- Preparing and distributing electronic ballots
+- Judging exception requests
+- Assisting candidates in preparing and sharing statements
+- Tallying voting results according to the rules in this charter
+
+### Eligibility to vote
+
+Anyone who has at least 50 contributions in the last 12 months is eligible to
```
[the remark this answers is not in the document]
Comment (msfussell) [reply steering-and-technical-committee-charter.md]: Updated

File: steering-and-technical-committee-charter.md (hunk 1 above)
Comment (msfussell) [reply steering-and-technical-committee-charter.md]: Updated

File: steering-and-technical-committee-charter.md (hunk 2 above)
Comment (msfussell) [reply steering-and-technical-committee-charter.md]: Updated

**Review by yaron2 — APPROVED** [review]
LGTM

**Comment by yaron2** [comment]
We have a steering committee :)

**Pull request merged**